civihr_employee_portal/templates/civihr-employee-portal-civihr-report-settings-age-group.tpl.php

```diff
@@ -1,75 +1,78 @@
-                    <thead>
-                        <tr>
-                        <?php
-                          foreach ($header as $field => $label):
-                            if (false) {
-                              continue;
-                            }
-                        ?>
-                            <th>
-                                <?php print $label; ?>
-                            </th>
-                        <?php endforeach; ?>
-                            <th><?php print t('Edit'); ?></th>
-                            <th><?php print t('Delete'); ?></th>
-                        </tr>
-                    </thead>
-                <?php endif; ?>
-                <tbody>
-                <?php foreach ($rows as $row_count => $row): ?>
-                    <tr>
-                        <?php
-                            foreach ($row as $field => $content):
-                                if ($field === 'id'):
-                                    continue;
-                                endif;
-                        ?>
-                            <td>
-				<?php print strip_tags(html_entity_decode($content)); ?>
-								<?php //if (_task_can_be_edited($row['id'])): ?>
-                                </a>
-								<?php //endif; ?>
-                            </td>
-                        <?php endforeach; ?>
-                            <td>
-                                <?php if ($canEdit): ?>
-                                <a  href="/reports/settings/age_group/nojs/edit/<?php print strip_tags($row['id']); ?>"
-                                    class="ctools-use-modal ctools-modal-civihr-custom-style ctools-use-modal-processed">
-				<?php endif; ?>
-                                    Edit
-                                <?php if ($canEdit): ?>
-                                </a>
-                                <?php endif; ?>
-                            </td>
-                            <td>
-                                <?php if ($canDelete): ?>
-                                <a  href="/reports/settings/age_group/nojs/delete/<?php print strip_tags($row['id']); ?>"
-                                    class="ctools-use-modal ctools-modal-civihr-custom-style ctools-use-modal-processed">
-				<?php endif; ?>
-                                    Delete
-                                <?php if ($canDelete): ?>
-                                </a>
-                                <?php endif; ?>
-                            </td>
-                    </tr>
-                <?php endforeach; ?>
-                </tbody>
-            </table>
-        </div>
+<div class="panel-pane pane-block chr_panel chr_panel--no-padding">
+  <div class="pane-content">
+    <div class="chr_search-result__header">
+      <div class="chr_search-result__total">
+        Results
+        <span class="chr_search-result__total__count"><?php echo count($rows); ?></span>
```

Comment:
Ok I understand the issue, and won't block the PR because of this. 
But I believe we need to plan some progressive refactorings to fix this kind of thing, this clearly tries to follow BEM, but it's [incorrect](http://getbem.com/faq/#css-nested-elements).